android/src/main/java/com/opentokreactnative/utils/EventUtils.java

```diff
@@ -99,6 +99,7 @@ public static WritableMap prepareAudioNetworkStats(SubscriberKit.SubscriberAudio
         audioStats.putInt("audioPacketsLost", stats.audioPacketsLost);
         audioStats.putInt("audioBytesReceived", stats.audioBytesReceived);
         audioStats.putInt("audioPacketsReceived", stats.audioPacketsReceived);
+        audioStats.putDouble("timeStamp", stats.timeStamp);
```

Comment:
can you rename `timeStamp` to `timestamp` please? So remain consistent with the documentation: https://tokbox.com/developer/sdks/js/reference/Subscriber.html#getStats

Thanks!